tests/jobs/replace_string/test_integration.py

```diff
@@ -8,16 +7,21 @@
 QUEUE_SYSTEM LOCAL
 NUM_REALIZATIONS 1
 FORWARD_MODEL COPY_FILE(<FROM>=<CONFIG_PATH>/file.txt, <TO>=file.txt)
-FORWARD_MODEL REPLACE_STRING(<FROM>={FROM}, <TO>={TO}, <FILE>=file.txt)
+FORWARD_MODEL REPLACE_STRING(<FROM>={FROM}, <TO>="{TO}", <FILE>=file.txt)
 """
 
 
-@pytest.mark.skipif(
-    sys.platform == "darwin", reason="Skip test for Mac OS - invalid use of sed"
-)
 @pytest.mark.parametrize(
     "input_text, replace_from, replace_to, expected",
-    [("something", "something", "else", "else"), ("1,2,3,4", "[0-9]", "N", "N,N,N,N")],
+    [
+        ("something", "something", "else", "else"),
```

Comment:
Could you also add an example for multiple occureses of the same sub-string?

Reply:
:+1:

Reply:
I changed the first example to contain multiple. The integration test is pretty slow